components/director/cmd/systemfetcher/main.go

```diff
@@ -0,0 +1,166 @@
+package main
```

Comment:
I may have missed it and it could have been discussed already - but what about e2e tests?